app/models/system_console.rb

```diff
@@ -53,10 +53,11 @@ def self.launch_proxy(remote_address, remote_port)
       return nil
     end
 
-    command = '/usr/bin/socat', "TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"
-    _log.info("Running socat proxy command: #{command.join(' ')}")
+    command = "/usr/bin/socat"
+    args = ["TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"]
+    _log.info("Running socat proxy command: #{args.join(' ')}")
```

Comment:
Probably should put command in the log line...in fact, this can be some cleaner with something like:

```ruby
command = AwesomeSpawn::CommandLineBuilder.new.build("/usr/bin/socat", ["TCP-LISTEN:#{local_port},fork", "TCP:#{remote_address}:#{remote_port}"])
_log.info("Running socat proxy command: #{command}")
pid = spawn(AwesomeSpawn::CommandLineBuilder.new.build(command, args))
```

WDYT?

Reply:
any string interpolation will cause brakeman to fail as Oleg's comment alludes to, so I'm going to have to go with, I don't like this, sorry.